Apps/WebAPI/sdLitica.WebAPI/sdLitica.WebAPI.csproj

```diff
@@ -13,8 +13,10 @@
     <PackageReference Include="Microsoft.AspNet.WebApi" Version="5.2.7" />
     <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.1.2" PrivateAssets="All" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.2-beta1" />
     <PackageReference Include="Swashbuckle.AspNetCore" Version="4.0.1" />
     <PackageReference Include="Swashbuckle.AspNetCore.Swagger" Version="4.0.1" />
+    <PackageReference Include="Vibrant.InfluxDB.Client" Version="4.0.1" />
```

Comment:
Please, do not reference ``InfluxDB`` packages in this project. Encapsulate it in a new project in the ``Persistance Layer``. The new project can be named as: ``sdLitica.TimeSeries``. Also, the implementation of the ``InfluxService`` should be in the new project.